src/sentinel.c

```diff
@@ -1516,6 +1536,22 @@ void rewriteConfigSentinelOption(struct rewriteConfigState *state) {
             rewriteConfigRewriteLine(state,"sentinel",line,1);
         }
 
+        /* sentinel rename-config */
+        if (master->config_command != SENTINEL_CONFIG_COMMAND) {
```

Comment:
since `master->config_command` could be set to NULL (see code change below), it's better to check for NULL first, before string comparison.